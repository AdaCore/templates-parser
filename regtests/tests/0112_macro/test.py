from test_support import *

run('print_tree', ["macro.tmplt"])
run('testme', ["macro.tmplt"])
