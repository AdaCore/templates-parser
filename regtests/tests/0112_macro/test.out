[INFO] macro.tmplt
[C_INFO]  1 FALSE
[TEXT] > macro for 8       : this text : 8   <
[TEXT] > macro for none    : this text :     <
[TEXT] > macro for xyz     : this text : xyz <
[TEXT] > macro for var ONE : @_MNAME(@_ONE_@)_@   <
> macro for 8       : this text : 8   <
> macro for none    : this text :     <
> macro for xyz     : this text : xyz <
> macro for var ONE : this text : 1   <
